### ISSUE TITLE
audio buffers half of the duration 

### ISSUE THREAD

**Issue by RekarBotany** [react-native-kit/react-native-track-player#121]
how can i make the buffering longer, a 6 minutes song will buffer 3 mins then buffers slowly which makes Seeking song slower

**Comment by Guichaguri** [comment]
You can set the buffering options in [setupPlayer](https://github.com/react-native-kit/react-native-track-player/wiki/Documentation#setupplayeroptions).

`minBuffer`: The minimum time in seconds that need to be buffered all the time.
`maxBuffer`: The maximum time in seconds that need to be buffered to stop loading more of the track.
`playBuffer`: The minimum time in seconds that need to be buffered to to start playing.

If it's below than `minBuffer`, it will start to buffer, after it reaches `playBuffer`, it will automatically resume the song, and it will stop constantly buffering when it reaches the `maxBuffer`.

Basically, `maxBuffer` shouldn't be too big or it will use a lot of memory, `minBuffer` shouldn't be too low or it will allow the user to start playing when there is not enough data to keep a constant playback, and `playBuffer` should be between `minBuffer` and `maxBuffer`.

**Comment by RekarBotany** [comment]
I tried this but doesnt affect anything on iOS

**Comment by phil-andrews** [comment]
I can confirm @RekarBotany these settings have no effect on iOS. Haven't checked Android.

```
  TrackPlayer.setupPlayer({
    minBuffer: 300,
    playBuffer: 20 ,
    maxBuffer: 300,
    maxCacheFiles: 100,
  }).then(() => {....
```

**Comment by sarbjeet-cmsDev** [comment]
@ntk860 have you found any solution to this? because I am trying to play around 160MB mp3 file

**Comment by ataravati** [comment]
@Guichaguri Any updates on this? This is the only reason I switched from react-native-sound to this library. If it loads the entire file before playing it, it's not actually streaming.

**Comment by varaprasadh** [comment]
Not sure why this thread is being closed. But is there a work around for iOS ?

**Comment by muhammadsmalik** [comment]
I'm still facing this issue. Seems like it hasn't been fixed yet. I'd appreciate any updates to this. Thanks.

**Comment by raymondjacobson** [comment]
This issue is still relevant
